networkit/stats.pyx

```diff
@@ -16,7 +16,19 @@ cdef extern from "<networkit/auxiliary/Parallel.hpp>" namespace "Aux::Parallel":
 
 def gini(values):
```

Comment:
Not sure if we need this feature at all, it's probably already implemented in other external packages. I think we should deprecate it.